tests/integration_test/data/single_app_as_job/test_ha_2_servers_after_first_round.yml

```diff
@@ -0,0 +1,14 @@
+system_setup: ./data/system/ha_2_servers_2_clients.yml
+apps_root_dir: ./data/apps
+single_app_as_job: True
+cleanup: True
+
+
+tests:
+  - app_name: np_sag
+    validators:
+      - path: tests.integration_test.validators.LogResultValidator
+        args: {
+          expected_in_result: "Round 1 started",
```

Comment:
should be `Round 2 started`? since we abort the job after round 1

Reply:
The first round is actually round 0, so it aborts after round 0 finishes. I was trying to do more with validations, but in practice, it doesn't really matter because the logs seem to be persisted across servers for the job now, and it is hard to do more advanced validation without finding timing of events and comparing that.